BlueprintUI/Sources/Layout/KeyboardObserver.swift

```diff
@@ -1,14 +1,14 @@
 //
```

Comment:
Changes in this file copied from Listable, in particular the `shared` instance, which ensures the keyboard frame is always being observed.